Automatic Validation ended with:
> 2024-08-14T18:09:16.5177446Z Manifest Validation Failed
 2024-08-14T18:09:21.5658770Z ##[error]Manifest Validation Failed

(Automated response - build 887.)

--- a/manifests/m/Maltego/Maltego/4.7.0/Maltego.Maltego.installer.yaml
+++ b/manifests/m/Maltego/Maltego/4.7.0/Maltego.Maltego.installer.yaml
@@ -11,7 +11,7 @@ AppsAndFeaturesEntries:
   ProductCode: Maltego
 Installers:
 - Architecture: x64
-  InstallerUrl: https://downloads.maltego.com/maltego-v4/windows/MaltegoSetup.v4.7.0.exe
-  InstallerSha256: 840ED0E868A525C9135EAD3DBA3C6C9BA487096DA4C1FEF77E8D88CC5125D473
+  InstallerUrl: https://downloads.maltego.com/maltego-v4/windows/MaltegoSetup.JRE64.v4.7.0.exe
+  InstallerSha256: 6CD600362BA4337DE4A2FA5EECB2028D156B270F1098C790B0FEEFD43FDF3DFB
 ManifestType: installer
 ManifestVersion: 1.6.0

--- a/manifests/m/Maltego/Maltego/4.7.0/Maltego.Maltego.installer.yaml
+++ b/manifests/m/Maltego/Maltego/4.7.0/Maltego.Maltego.installer.yaml
@@ -9,6 +9,11 @@ UpgradeBehavior: uninstallPrevious
 AppsAndFeaturesEntries:
 - DisplayVersion: '4'
   ProductCode: Maltego
+- # There's an Eclipse JRE bundled in the installer wizard
+  DisplayName: Eclipse Temurin JRE with Hotspot
+  DisplayVersion: 17.0.8.101
+  Publisher: Eclipse Adoptium
+  ProductCode: '{85D3BA14-41B0-4B9B-B01F-A594869B2C9A}'
 Installers:
 - Architecture: x64
   InstallerUrl: https://downloads.maltego.com/maltego-v4/windows/MaltegoSetup.JRE64.v4.7.0.exe

